libs/sdk-ui-dashboard/src/presentation/dragAndDrop/draggableAttributeFilter/DraggableAttributeFilterPlaceholder.tsx

```diff
@@ -0,0 +1,42 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { useDashboardDrag } from "../useDashboardDrag";
+import classNames from "classnames";
+import { useDashboardSelector } from "../../../model";
+import { selectIsInEditMode } from "../../../model/store/ui/uiSelectors";
+
+export type CustomDashboardAttributeFilterPlaceholderComponentProps = {
```

Comment:
Should these be in a types.ts file instead? Do they need some TSDoc? Or are they just some internal thing?